registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java

```diff
@@ -442,6 +446,8 @@ public void consumerHealthCheck(Handler<HealthCheckDTO> eventHandler, String add
 				healthCheckDTO.setEventBusConnected(false);
 				healthCheckDTO.setFailureReason(f.cause().getMessage());
 			}
+			logger.debug("Consumer health check ended {} {}",

```

Comment:
why do we have two {} ?

Reply:
can we also pass the result from f.succeeded() in the log?

Reply:
Fixed and added healthCheckDTO.isEventBusConnected() so not removed { }
We cant pass result of  f.succeeded() in log  because  it is async of type Map<String,List<PartitionInfo>>